scipy/spatial/transform/rotation.py

```diff
@@ -281,3 +306,113 @@ def as_rotvec(self):
             return rotvec[0]
         else:
             return rotvec
+
+    @classmethod
+    def _elementary_quat_compose(cls, seq, angles, intrinsic=False):
+        # Initialize result to first axis
+        result = _make_elementary_quat(seq[0], angles[:, 0])
+
+        for idx, axis in enumerate(seq[1:], start=1):
+            if intrinsic:
+                result = _compose_quat(
+                    result,
+                    _make_elementary_quat(axis, angles[:, idx]))
+            else:
+                result = _compose_quat(
+                    _make_elementary_quat(axis, angles[:, idx]),
+                    result)
+        return result
+
+    @classmethod
+    def from_euler(cls, seq, angles, degrees=False):
+        """Initialize rotation from Euler angles.
+
+        Parameters
+        ----------
+        seq : str
+            `seq` must be a string of upto 3 characters belonging to the set
+            {'x', 'y', 'z'} or {'X', 'Y', 'Z'}. Each lowercase character
+            represents an extrinsic rotation around the corresponding axis and
+            an uppercase character represents an intrinsic rotation. Extrinsic
+            and intrinsic rotations cannot be mixed in one function call.
+        angles : float or array_like, shape (N, ) or (N, [1 or 2 or 3])
```

Comment:
I think as you explain possible cases below, you can simply write `array_like`. But I'm fine with your description as well. As you prefer.